dotCMS/src/main/java/com/dotcms/util/TimeUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.dotcms.util;
+
+import com.dotmarketing.util.Config;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.util.Config'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)